### PR TITLE
ESA-72 & autotoolize

### PR DESCRIPTION
**Pull request by vguzmanp**

### CODE REVIEW

File: AssemblyInfo/AssemblyInfo.cs [hunk 1]
```diff
@@ -1,45 +1,54 @@
...
-// and "{Major}.{Minor}.{Build}.*" will update just the revision.
+// The assembly version has following format :
+//
+// Major.Minor.Build.Revision
+//
+// You can specify all values by your own or you can build default build and revision
+// numbers with the '*' character (the default):
 
-[assembly: AssemblyVersion ("1.0.*")]
+[assembly: AssemblyVersion("0.2")]
```
Comment (vmartos) [inline AssemblyInfo/AssemblyInfo.cs]: This assemblyVersion is set to 0.2 for any reason? It's changed during build time by the Longomatch release tag?

File: build/build_env.mk [hunk 2]
```diff
@@ -0,0 +1,8 @@
+LM_DEPS = \
```
Comment (vmartos) [inline build/build_env.mk]: Maybe should be VAS_DEPS, as we are in the VAS repo?

**Comment by vmartos** [comment]
Code review Done

File: AssemblyInfo/AssemblyInfo.cs (hunk 1 above)
Comment (vguzmanp) [inline AssemblyInfo/AssemblyInfo.cs]: I set it to 0.2 as a placeholder. This way we avoid using the git-version-gen which, without tags, would make the version `UNKNOWN`

File: build/build_env.mk (hunk 2 above)
Comment (vguzmanp) [inline build/build_env.mk]: Actually it's never used in VAS, we could remove it.
At the beginning I thought of using the VAS `build` folder and symlink to it from LongoMatch, but it's probably easier to have a copy

**Comment by vguzmanp** [comment]
Ready to review again

**Comment by vmartos** [comment]
:+1:

**Pull request merged**